### ISSUE TITLE
Provide Grafana + Omniperf plugin as Docker image in a public registry like Dockerhub

### ISSUE THREAD

**Issue by ELCapitanLLNL** [AMDResearch/omniperf#418]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
I would like do be able to pull an image of grafana with the omniperf plug-in so can I can `docker pull omniperf-grafana` as an example.

**Describe alternatives you've considered**
Currently I am using the Dockerfile provided in the repo 
in https://github.com/ROCm/omniperf/blob/amd-staging/grafana/Dockerfile
However, I need to modify it to add my secret sauce, which makes it more difficult to maintain.

**Additional context**
Currently other omniperf images are being hosted under Mr. Ramos account at
https://hub.docker.com/u/colramos

**Comment by coleramos425** [comment]
This should be a straightforward task. I'll need to confirm with the project PM to sign off on publishing this container. Expect an update shortly...

**Comment by ELCapitanLLNL** [comment]
hi @coleramos425 , any update on this straightforward task?  thank you!

**Comment by coleramos425** [comment]
CC: @njobypet 

For approval on container publishing

**Comment by njobypet** [comment]
grafana with the omniperf plug-in is a good idea. Can we use Rocprof-compute instead of Omniperf ?

**Comment by coleramos425** [comment]
> grafana with the omniperf plug-in is a good idea. Can we use Rocprof-compute instead of Omniperf ?

Yes we can, we'll just need to wait for #428 to merge. @njobypet I am creating a subtask in Xuan's PR to make sure this is included.

**Comment by njobypet** [comment]
@coleramos425 , Please check grafana licensing requirements as well. Just to ensure that we are good to re-distribute.

**Comment by coleramos425** [comment]
Unfortunately, that's outside of my domain @njobypet. You'll need to reach out to the legal team. Licencing info can be found [here](https://github.com/ROCm/omniperf/tree/amd-staging/grafana/plugins/omniperf_plugin). I'll note that in Xuan's ticket.

**Comment by ELCapitanLLNL** [comment]
Happy Monday!  
Any update on this now that the rename happened.
Thanks!

**Comment by coleramos425** [comment]
> Happy Monday! Any update on this now that the rename happened. Thanks!

CC: @njobypet

**Comment by amd-hsivasun** [comment]
Imported to ROCm/rocm-systems
